src/styles/vars.scss

```diff
@@ -31,3 +31,7 @@ $collaboration-gold: #ffd700;
 $collaboration-diamond: #b9f2ff;
 
 $hoverTransitionDuration: 0.25s;
+
+$navbarHeight: 67px;
```

Comment:
Chyba wrzuciłbym to lokalnie a nie globalnie

Reply:
zmienione w https://github.com/akai-org/akai-webpage/pull/45/commits/22d5e01b653310229cfd625b53ac215bb19773eb

Reply:
dobry pomysł, ale patrz case niżej 😄